test/index.js

```diff
@@ -77,6 +77,13 @@ function args() {
   return arguments;
 }
 
+//  arrayToList :: Array a -> List a
+function arrayToList(xs) {
+  let list = Nil;
+  for (let idx = 0; idx < xs.length; idx += 1) list = Cons (xs[idx], list);
+  return list;
+};
```

Comment:
Let's use an arrow function.

We use function declarations below to enable us to verify the correctness of implementations by checking `arguments.length` (`this.map (f)` is not an acceptable implementation of `Array#fantasy-land/map` due to the extra arguments). Such assertions are not required here, so we can use an arrow function. :)